### PR TITLE
Add config for gravityChain pull Size and modify go.mod #88

### PR DESCRIPTION
**Pull request by lzxm160**
work on #88

### CODE REVIEW

File: committee/committee.go [hunk 1]
```diff
@@ -51,6 +51,7 @@ type Config struct {
 	CacheSize                  uint32   `yaml:"cacheSize"`
 	NumOfFetchInParallel       uint8    `yaml:"numOfFetchInParallel"`
 	SkipManifiedCandidate      bool     `yaml:"skipManifiedCandidate"`
+	GravityChainSize           uint64   `yaml:"gravityChainSize"`
```
Comment (zjshen14) [inline committee/committee.go]: GravityChainBatchSize?

File: committee/committee.go [hunk 2]
```diff
@@ -146,6 +148,10 @@ func NewCommittee(kvstore db.KVStore, cfg Config) (Committee, error) {
 	if cfg.NumOfFetchInParallel > 0 {
 		fetchInParallel = cfg.NumOfFetchInParallel
 	}
+	gravityChainSize := uint64(10)
```
Comment (zjshen14) [inline committee/committee.go]: I think you want to make it to 1000.

Here 1000 is the eth block gap, and every 100 eth block gap, we will sample election data once, thus resulting into 10 election data points. cc @CoderZhi 

Can you experiment again with 1000? How long does a node need to start?

File: go.mod [hunk 3]
```diff
@@ -16,11 +16,12 @@ require (
 	github.com/golang/mock v1.3.1
 	github.com/golang/protobuf v1.3.1
 	github.com/google/pprof v0.0.0-20190515194954-54271f7e092f // indirect
+	github.com/iotexproject/go-p2p v0.2.10
```
Comment (zjshen14) [inline go.mod]: Curious why election now needs to depend on P2P library. Can you help to investigate it too?

Comment (lzxm160) [reply go.mod]: sure

**Pull request merged**